tests/integration/network/test_node_storage.py

```diff
@@ -46,6 +46,6 @@ def start_lonely_learning_loop():
 
     yield deferToThread(start_lonely_learning_loop)
 
-    assert list(newcomer.known_nodes)
+    assert list(node.mature() for node in newcomer.known_nodes)
```

Comment:
Is L51 the one that fails? This one asserts that there are known_nodes only?

Reply:
> Is L51 the one that fails?

Yes

> This one asserts that there are known_nodes only?

Yep